doc/circulationrules.md

```diff
@@ -1,5 +1,25 @@
 # Circulation Rules
 
+<!-- ../../okapi/doc/md2toc -l 2 -h 3 circulationrules.md -->
```

Comment:
What is this line for?

Reply:
Paste it into a shell and run it. Its output is an up-to-date table of content. Copy the output and paste it into this file.
https://github.com/folio-org/okapi/blob/master/doc/md2toc

Reply:
Ok, please can a comment be included to explain that this command is used to generate a new table of contents. 

(The path assumes that users have Okapi cloned two directories above this directory)